be/src/common/process_exit.cpp

```diff
@@ -29,6 +31,11 @@ std::atomic<bool> k_starrocks_exit;
 // but also waiting for all threads to exit gracefully.
 std::atomic<bool> k_starrocks_quick_exit;
 
+// NOTE: when backend is going to shutdown, if FE's heartbeat request received, the status code
+// in the response will be set to SHUTDOWN, then FE leader will know the node is in shutdown immediately.
+// This flag records whether there is a such response with status SHUTDOWN sent back to FE.
+std::atomic<bool> k_starrocks_fe_heartbeat_aware_shutdown;
```

Comment:
```suggestion
std::atomic<bool> k_starrocks_fe_heartbeat_aware_shutdown{false};
```
Better to assign an initial value explicitly.

Reply:
done